### PR TITLE
Updates to css

### PR DESCRIPTION
**Pull request by mortonjt**
Depends on #434 

Also having a minor issue with the horizontal scrollbar.  It seems to work once the console is opened up.

Todo
- [x] Fix the css coloring
- [x] Fix the horizontal scrollbar issue
- [x] Fix the vertical scrollbar issue
- [ ] Fix elements disappearing from the grid (happens upon resize on the tabs that are not on-focus).
- [ ] Fix double clicking scheme
- [ ] Fix missing tick.png in ipython notebook

### CODE REVIEW

**Comment by mortonjt** [comment]
Also want to get rid of vertical scrollbar.  We want to make the headers fixed

**Comment by antgonza** [comment]
Failures ...

**Comment by antgonza** [comment]
@mortonjt could you pull from the new-api branch and fix the failures? Thanks!

**Comment by mortonjt** [comment]
Working on it :L

**Comment by mortonjt** [comment]
That should fix the tests - all widths are now at 100%, removing the scrollbar.

Still can't fix the vertical scrollbar issue ...

File: emperor/support_files/js/controller.js [hunk 1]
```diff
@@ -32,8 +32,8 @@ EmperorController = function(dm, divId){
 
   // Constants
   this.GRID_SCALE = 0.97;         // Scaling constant for grid dimensions
-  this.SCENE_VIEW_SCALE = 0.5;   // Scaling constant for scene plot view dimensions
-  this.SLICK_WIDTH = 25;         // Constant for width in slick-grid
+  this.SCENE_VIEW_SCALE = 0.5;    // Scaling constant for scene plot view dimensions
```
Comment (wasade) [inline emperor/support_files/js/controller.js]: would it ever be useful to source these from a config file?

**Comment by antgonza** [comment]
@mortonjt let us know if you need help here, you could fix the conflicts, merge this  and we could split the rest of the list ...

**Comment by mortonjt** [comment]
That should fix the merge conflicts.

There is some css magic that I'm a bit confused about with setting the heights in the `EMP-xxxxxx` headers.  Perhaps we can discuss this after merging this?

**Comment by ElDeveloper** [comment]
@mortonjt, thanks but it seems like there's a problem, when looking at the example I get this error:

```
THREE.WebGLRenderer 73
require-2.1.22.min.js:19 ReferenceError: EmperorViewControllerABC is not defined(…)ea.check @ require-2.1.22.min.js:19
view-controller.js:376 Uncaught TypeError: Cannot read property 'setColumns' of undefined
```

Not sure what might be going on, are you able to see the plot on your computer?

**Comment by antgonza** [comment]
Same here, my guess is that you didn't update the calls to required ...

**Comment by mortonjt** [comment]
That should do it.  Thanks!

**Comment by mortonjt** [comment]
I'm not seeing the missing tick.png error.  Can one of you reduplicate this error?

**Comment by antgonza** [comment]
AFAIL The missing image only happens in ipython ...

**Comment by ElDeveloper** [comment]
@antgonza is right, the way to fix this would be to load through CSS
like it is done in jQuery or chosen.

**Comment by mortonjt** [comment]
Going to close this PR.  This doesn't look like this is going to be easily recoverable ...

**Comment by ElDeveloper** [comment]
@mortonjt just a friendly reminder on this PR.

**Pull request closed without merging**